### PR TITLE
Update .travis.yml to use yarn

### PR DESCRIPTION
**Pull request by sadafie**
In the previous PR, I accidentally forgot to update `.travis.yml` to use `yarn` not `npm` commands. I'm not sure this will have any visible side-effects right now, but it'll probably prevent random build issues / bugs in the future.

Fixes https://meetup.atlassian.net/browse/WP-236

### CODE REVIEW

**Comment by sadafie** [comment]
Hmmm, Prelim until I figure out why the travis builds didn't run.

**Comment by sadafie** [comment]
Cool, builds started, just very delayed for some reason.

File: .travis.yml [hunk 1]
```diff
@@ -40,9 +40,9 @@ after_success:
   - echo "NPM_TAG=$NPM_TAG"
   - >
     [ "${TRAVIS_BRANCH}" = "master" ] &&
-    npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
+    yarn version --no-git-tag-version --new-version $GIT_TAG &&
```
Comment (mmcgahan) [inline .travis.yml]: Seems like it's a good idea to make a new git tag? Tags are used by GitHub to track releases: https://github.com/meetup/meetup-web-platform/releases

Comment (nlaz) [inline .travis.yml]: I've been using the new beta versions to test features across repos and I've been checking the GitHub releases/tags to know which versions are available. This has been useful to me personally, but I'm also not sure if there is a better way to find the latest beta version for a repo.

Comment (mmcgahan) [inline .travis.yml]: @nlaz you can also ask NPM directly for which versions are available, although it's not quite as descriptive as GitHub's 'releases' list

```sh
$ npm view meetup-web-platform versions
```

To specifically find the latest beta version, use

```
$ npm view meetup-web-platform@beta version
1.0.711-beta
```

Comment (sadafie) [inline .travis.yml]: > Seems like it's a good idea to make a new git tag? Tags are used by GitHub to track releases: https://github.com/meetup/meetup-web-platform/releases

Sorry, I'm just seeing this. So yarn will make it's own git tags, but i'm assuming we want to use our own defined tags which is what the  `--new-version $GIT_TAG` argument (should) do. 

I'm circling back to see why this is breaking the build, but also, @mattkime or anyone else --- any suggestions on how to test a change like this? Thanks!

Comment (sadafie) [inline .travis.yml]: Ah I see. It jumped into an interactive mode with `yarn publish`. I need to look to see how to by pass that with CLI arguments.

Comment (sadafie) [inline .travis.yml]: https://github.com/yarnpkg/yarn/issues/2445
non-interactive mode on `yarn publish` doesn't exist. I will revert this to npm publish for now unless anyone has concerns. cc @mattkime @mmcgahan

**Comment by sadafie** [comment]
turns out `yarn publish` is interactive, which doesn't work in scripts. there's already an issue filed on yarn. and I don't love mixing npm and yarn commands, so I'm going to go ahead and close this.

**Pull request closed without merging**